app/views/categories/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= form_with url: categories_path, model: @group do |f| %>
+  <header class="header">
```

Comment:
* As the previous reviewer pointed out that an error that occurs when trying to save a category still persists Kindly fix it 🙏 
![image](https://user-images.githubusercontent.com/52497246/144712710-01b1f756-9a15-482a-b9ef-21f7282a1a02.png)

Reply:
![image](https://user-images.githubusercontent.com/54015740/144716432-ed76ec2d-9040-4871-8b9c-acb21210a10f.png)
fixed it

Reply:
![desktop](https://user-images.githubusercontent.com/54015740/144716903-8c69b721-43d0-42f5-a466-c0c34533a86f.png)